jedi/_compatibility.py

```diff
@@ -500,3 +501,16 @@ class Parameter(object):
         VAR_POSITIONAL = object()
         KEYWORD_ONLY = object()
         VAR_KEYWORD = object()
+
+
+class GeneralizedPopen(subprocess.Popen):
+    def __init__(self, *args, **kwargs):
+        creation_flags = 0
+        if os.name == 'nt':
+            try:
+                # Was introduced in Python 3.7.
+                CREATE_NO_WINDOW = subprocess.CREATE_NO_WINDOW
+            except AttributeError:
+                CREATE_NO_WINDOW = 0x08000000
+            creation_flags = CREATE_NO_WINDOW
```

Comment:
We can directly do `kwargs[ 'creation_flags' ] = CREATE_NO_WINDOW` instead of using a variable.

Reply:
Thanks